standalone/src/models/product-selection/product-of-selection/builders.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { ProductOfSelectionGraphql } from './index';
+
+describe('ProductOfSelection Builder', () => {
+  it('should build properties for the GraphQL representation', () => {
+    const graphqlModel = ProductOfSelectionGraphql.random().build();
+
+    expect(graphqlModel).toEqual(
+      expect.objectContaining({
+        productRef: expect.objectContaining({
+          __typename: 'Reference',
+          typeId: 'product',
+          id: expect.any(String),
```

Comment:
No need to check these properties. The `__typename` one is already enough.

Reply:
But why is that? I understand we don't need to test those when we are populating a field with a submodel cause we assume that those fields are already tested in the submodel and we just want to make sure the `__typename` is correct, but this is the test related to the submodel itself, and if we remove those fields how can we be confident that those fields even exist in this submodel?

Reply:
Ahhh, you're right! This is the fields of another submodel, thanks for noticing.